tests/integration_tests/reports/utils.py

```diff
@@ -83,3 +95,99 @@ def insert_report_schedule(
     db.session.add(report_schedule)
     db.session.commit()
     return report_schedule
+
+
+def create_report_notification(
+    email_target: Optional[str] = None,
+    slack_channel: Optional[str] = None,
+    chart: Optional[Slice] = None,
+    dashboard: Optional[Dashboard] = None,
+    database: Optional[Database] = None,
+    sql: Optional[str] = None,
+    report_type: Optional[ReportScheduleType] = None,
+    validator_type: Optional[str] = None,
+    validator_config_json: Optional[str] = None,
+    grace_period: Optional[int] = None,
+    report_format: Optional[ReportDataFormat] = None,
+    name: Optional[str] = None,
+    extra: Optional[Dict[str, Any]] = None,
+    force_screenshot: bool = False,
+) -> ReportSchedule:
+    report_type = report_type or ReportScheduleType.REPORT
```

Comment:
I would likely inline `report_type` and `target` as they are single use variables.

Reply:
Good call. I originally just copied this code from another file.